sdk/servicebus/azure-messaging-servicebus-stress/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM maven:3.8.6-openjdk-11 as builder
+FROM stresspgs7b6dif73rup6.azurecr.io/java-tools/mvn:java11-mvn3.8.8 as builder
```

Comment:
We should probably put this in the azsdkengsys acr? Or mirror it into the ACR used for the weekly prod deploys.

Reply:
We (non-eng folks) don't have push access to azsdkengsys (at least I don't, I do have pull access though).
So we'd need to kindly ask you and other eng friends to push it for us every once in a while. Would you do?

Reply:
@lmolkova I think we can follow a similar model here as we do for net. If you want to copy from https://github.com/Azure/azure-sdk-for-net/tree/main/eng/containers I can set up a pipeline with publish permissions and we can use that?

Reply:
Sounds awesome! I updated the PR to implement it.